project.nix

```diff
@@ -14,11 +14,19 @@ proj = kpkgs.rp.project ({ pkgs, hackGet, ... }: with pkgs.haskell.lib;
         sed -i -e 's/^\(test-suite\|benchmark\) /executable /' -e '/^ *type: *exitcode-stdio-1.0$/d' *.cabal
       '';
     });
+  # Nixpkgs with the `find-libs` command added as a top-level package
+  nixpkgsWithFindLibs = import (pkgs.fetchFromGitHub {
+    owner = "ryantrinkle";
+    repo = "nixpkgs";
+    rev = "a9a141a10f32691228ae7668e571359f2aaf82e4";
+    sha256 = "0ncqzsq3rwkh4g68bxsxmznq4z4pcqy6wclslnrsnqbq1b2lk2gl";
+  }) {};
   in {
     name = "chainweb";
     overrides = self: super: {
       chainweb = enableCabalFlag (
         justStaticExecutables (enableDWARFDebugging (convertCabalTestsAndBenchmarksToExecutables super.chainweb))) "use_systemd";
+      chainweb-portable = nixpkgsWithFindLibs.callPackage ./make-portable.nix {} "/opt/chainweb" self.chainweb;
```

Comment:
You can use `$ORIGIN` instead of `/opt/chainweb` here so that ld.so will look for libraries in the same directory as the executable. That way, the libraries don't have to be in `/opt/chainweb`.

Reply:
Yep, I originally used "$ORIGIN/.." as the prefix, but that didn't work due to the ld-linux issues; since I needed a prefix anyway, I just kept it consistent.

Reply:
Oh, using `$ORIGIN` would be much nicer IMO.